index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" type="text/css" />
+  </head>
+
+  <body>
+    <main id= "flexContainer">
+
+      <div id="leftHalf">
+        <h1>Weather Report</h1>
+        <p>🌎 City of <span id="city">Ada</span></p>
+
+        <section id="tempFeature">
+          <h2 id="tempCount">Temperature: <span id="num">0</span></h2>
```

Comment:
Consider initializing with a class so that 0 is the correct color before the first time you press the up or down button.